src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -2207,6 +2210,118 @@
     crafted: true
 """
 
+helps['webapp sitecontainers create'] = """
+type: command
+short-summary: Create sitecontainers for a linux webapp
+long-summary: |
+    Multiple sitecontainers can be added at once by passing arg --sitecontainer-spec-file, which is the path to a json file containing an array of sitecontainer specs.
+    Example json file:
+    [{
+        "name" : "firstcontainer",
+        "image": "myregistry.io/firstimage:latest",
+        "target_port": "80",
+        "is_main": true,
+        "environment_variables" : [
+            {
+                "name" : "hello",
+                "value": "world"
```

Comment:
Environment variable must be a APPSETTING reference, lets look some text which looks like an AppSetting

APPSETTING_KEY